packages/codegen-ui-react/lib/helpers/index.ts

```diff
@@ -113,3 +113,17 @@ export function isAliased(componentType: string): boolean {
 export function removeAlias(aliasString: string): string {
   return aliasString.replace(/(Custom$)/g, '');
 }
+
+export function getControlledComponentDefaultValue(
```

Comment:
I'm confused on this function. Can't a lot of components be controlled, but this one is only looking for TextField components?

Reply:
There are other controlled components (like Toggle and Slider) but those seem to still support defaultValue as a component param.  Since I converted TextField to a controlled component, this is a direct follow up to that change which I confirmed does not support defaultValue.